plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -955,6 +956,217 @@ public void testSettingChangeDataFeedEnabledProperty()
                 .contains("change_data_feed_enabled = true");
     }
 
+    @Test
+    public void testHighlyNestedData()
+    {
+        // TODO consider moving this in BaseConnectorTest
```

Comment:
You could put these tests in a base class in io.trino.testing and then run it for the relevant connectors and file formats by adding derived classes (e.g. BaseOrcWithBloomFiltersTest)

Reply:
I am considering moving tests in follow-up PR.

https://github.com/trinodb/trino/pull/17085#discussion_r1171225365

Reply:
Great finding Raunaq.

Reply:
https://github.com/trinodb/trino/pull/17414